tests/framework/extensions/rbac/constants.go

```diff
@@ -0,0 +1,46 @@
+package rbac
+
+const (
+	Admin                            = "admin"
+	RestrictedAdmin                  = "restricted-admin"
+	StandardUser                     = "user"
+	RoleReadOnly                     = "read-only"
+	RoleOwner                        = "cluster-owner"
```

Comment:
Good to specify full word :RoleClusterOwner ,like you specified for project.

Same for RoleMember :RoleClusterMember